src/test/api/karate/cards/cardsReminder.feature

```diff
@@ -32,46 +32,123 @@ Feature: CardsReminder
     ]
     """
 
-Scenario: ResetCardsReadsAndAcks
+    
+  * def entity1Array =
+      """
+      [   "ENTITY1_FR"
+      ]
+      """
+
+            * def entity2Array =
+      """
+      [   "ENTITY2_FR"
+      ]
+      """
+
+Scenario: Send card and receive reminder
+
+
+#Create new perimeter
+* call read('../common/createPerimeter.feature') {perimeter: '#(perimeter)', token: '#(authTokenAdmin)'}
+
+#Attach perimeter to group
+    Given url opfabUrl + 'users/groups/ReadOnly/perimeters'
+    And header Authorization = 'Bearer ' + authTokenAdmin
+    And request perimeterArray
+    When method patch
+    Then status 200
+
+
+    * def currentDate = new Date()
+    * def now = new Date().valueOf()
+    * def yesterday = new java.math.BigDecimal(now - 8640000)
+    * def tomorrow = new java.math.BigDecimal(now + 8640000)
+
+    * def hours = new java.lang.Integer(new Date().getHours())
+    * def minutes = new java.lang.Integer(new Date(now + 2*60*1000).getMinutes())
+    * def seconds = new java.lang.Integer(new Date().getSeconds())

```

Comment:
As for minutes it has to be the same for hours otherwise 9:59 will ends up to 9:01 


```suggestion
    * def datePlus2Min = new java.lang.Integer(new Date(now + 2*60*1000))
    * def hours = new java.lang.Integer(datePlus2Min.getHours())
    * def minutes = new java.lang.Integer(datePlus2Min.getMinutes())
    * def seconds = new java.lang.Integer(datePlus2Min.getSeconds())
 
```